### DISCUSSION TITLE
Pull request to validate questions

### DISCUSSION THREAD

**Discussion by shhitzelectric** [community/community#84998] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

Just wondering how it works is it in a queue to be looked over and reviewed or do I have to ask you

**Comment by Piero24** [comment]
Hi @shhitzelectric Yes the message is visible immediately without problems if you mean regarding your question above

**Reply by shhitzelectric** [reply]
I have a pull request on Main and I’m just wondering is it visible and in Que will somebody be able to look at it and validate it or help me with it or when do they validate?

**Reply by shhitzelectric** [reply]
Hi @Piero24 I have a pull request on Main and I’m just wondering is it visible and in Que will somebody be able to look at it and validate it or help me with it or when do they validate

**Comment by shhitzelectric** [comment]
I have a pull request on Main and I’m just wondering is it visible and in
Que will somebody be able to look at it and validate it or help me with it
or when do they validate

**Comment by Piero24** [comment]
Oh you mean on a project. Is this the repo: https://github.com/shhitzelectric/token-list? If you sent the pull request to the original repository then you just have to wait for someone to look at it and approve it or not... However, looking at the specific pull requests of the original repository I see that a couple of your pulls have been rejected. Check if it's the same. I'll attach an image for you
<img width="1416" alt="Screenshot 2024-01-10 at 00 18 02" src="https://github.com/community/community/assets/32714047/a2d275e1-ee71-4793-8f31-dd9d5786f673">

**Comment by shhitzelectric** [comment]
Yes, that's it. And this is a new pull after I deleted the repository, I
made a new pull request and after I submitted the first one it was looked
over within a couple hours. This one has been sitting for over two days.
Just wondering if there was a time or a day that they get reviewed

**Comment by Piero24** [comment, marked as answer]
@shhitzelectric If it is under review you just have to wait for it to be reviewed. It depends on the team and how many pull requests they have to check... each team has its own times.

**Reply by ghostinhershell** [reply]
Thanks so much for jumping in here and lending a hand @Piero24! 🚀

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @shhitzelectric! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by shhitzelectric** [reply]
Thank you

**Comment by shhitzelectric** [comment]
Thank you for your response.

**Comment by dilip478** [comment]
PLZ FREE USE

**Comment by shhitzelectric** [comment]
What??

**Comment by shhitzelectric** [comment]
Can somebody help me with getting a review on # 1359 pull request To
validate shhitzelectric (shh) Not too sure how to get a reviewer, but
looking to move forward with this project any help will help. Thank you.
